### PR TITLE
PLAT-1013: ZenTable unit tests

### PR DESCRIPTION
**Pull request by mplesec**
[JIRA ticket](https://reciprocitylabs.atlassian.net/browse/PLAT-1013)

#### Description
Some simple unit tests for these components since there's not much else to test yet.

#### ChangeLOG

- [ ] I have added all notable deployment/development environment (onprem!) changes to [CHANGELOG.md](https://github.com/reciprocity/zen-ui/blob/main/README.md)

#### Useful links
[Zen UI readme](https://github.com/reciprocity/zen-ui/blob/main/README.md)

[PR review guidelines](https://github.com/reciprocity/zengrc/blob/develop/doc/dev_environment_and_process/pull_request_reviews.md)

[Git commit guidelines](https://github.com/reciprocity/zengrc/blob/develop/doc/dev_environment_and_process/commit_guidelines.md)

### CODE REVIEW

File: src/components/zen-table-cell/test/zen-table-cell.spec.tsx [hunk 1]
```diff
@@ -1,8 +1,21 @@
...
 
 describe('zen-table-cell', () => {
   it('renders', async () => {
-    // TODO:
+    const page = await newSpecPage({
+      components: [ZenTableCell],
+      html: `<zen-table-cell>Content</zen-table-cell>`,
+    });
+
+    expect(page.root).toEqualHtml(`
```
Comment (chavlji) [inline src/components/zen-table-cell/test/zen-table-cell.spec.tsx:11]: I'm not sure regarding these snapshot tests... they're not very useful and are very hard to maintain.
We've deleted them from all other components.

**Review by chavlji — CHANGES_REQUESTED** [review]

File: src/components/zen-table-cell/test/zen-table-cell.spec.tsx (hunk 1 above)
Comment (mplesec) [reply src/components/zen-table-cell/test/zen-table-cell.spec.tsx:11]: I don't know about other component, but in these components they're necessary to test that the slot content is passed in as expected, and to check that an attribute was applied correctly. AFAIK there's no other way to determine correctness than this (apart from visual tests, but those are mean't to mostly test how components look).

Comment (chavlji) [reply src/components/zen-table-cell/test/zen-table-cell.spec.tsx:11]: Aha ok. Agree to that. I think @f-jauregui also had a case where he wanted to check if slot gets rendered.

**Review by chavlji — APPROVED** [review]

File: src/components/zen-table-cell/test/zen-table-cell.spec.tsx (hunk 1 above)
Comment (f-jauregui) [reply src/components/zen-table-cell/test/zen-table-cell.spec.tsx:11]: It's merged, but 🤷‍♂️ 

Shadow DOM of the component can be accessed by the `page.root.shadowRoot` object. 

To check if the slot was rendered:

```js
expect(page.root.shadowRoot.querySelector('slot')).toBeTruthy();
```

To check if some content was applied to a named slot:

```js
expect(page.root.querySelector('[slot="namedSlot"]')).toBeTruthy();
```

And so on...

Comment (mplesec) [reply src/components/zen-table-cell/test/zen-table-cell.spec.tsx:11]: Addressed in https://github.com/reciprocity/zen-ui/pull/135

**Pull request merged**